### ISSUE TITLE
Would you consider renaming this tool or making it compatible with the one in Debian?

### ISSUE THREAD

**Issue by sto** [krakozaure/tmpl#2]
Hi, first of all let me tell you that I find this tool quite useful, so thanks for writing it!

As I've been using it lately I decided that including it on the debian repository would be a good idea, but after filling the [ITP](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1043027) I noticed that there is already a tool with the same name on the archive.

I could package it with a different name for Debian, but if you change the name upstream we would be in sync.

Just think about it, I guess a name like `gotmpl` would also be good and a little bit clearer, but any name that does not create conflicts is good for me.

If you don't want to rename your tool it's OK, but I will have to do it for the debian package, so in any case I would like your input on the name to use.

**Comment by andrewshadura** [comment]
Alternatively, how about adding support for command-line options offered by [that other tmpl tool](https://github.com/benbjohnson/tmpl)?

```
$ echo '["foo","bar","baz"]' > tmpldata
$ tmpl -data=@tmpldata my.tmpl
```

**Comment by sto** [comment]
> Alternatively, how about adding support for command-line options offered by [that other tmpl tool](https://github.com/benbjohnson/tmpl)?
> 
> ```
> $ echo '["foo","bar","baz"]' > tmpldata
> $ tmpl -data=@tmpldata my.tmpl
> ```

yes, that would work too, sure, in fact if you do that we can replace the other tool

**Comment by sto** [comment]
Although I'm not really a golang programmer I've taken the liberty to create a [PR](https://github.com/krakozaure/tmpl/pull/3) to make your tool compatible with the one in Debian.

If you integrate my changes or an updated version of them in your repository I'll be able to package your upstream source as is because it provides a full replacement of the old command.

**Comment by sto** [comment]
Any news about this? Have you given it any thought?
I'm waiting for an answer to create and upload the package to Debian.
